src/parser/command.rs

```diff
@@ -323,16 +321,15 @@ pub fn parse_pragma<'a>(input: ParserInput<'a>) -> ParserResult<'a, Instruction>
 }
 
 /// Parse a pulse instruction, **including the PULSE keyword**.
```

Comment:
My fault - but this comment is out of date and should read like all the others. PULSE keyword is no longer included.

Reply:
Fixed!